modules/formulize/templates/admin/home.html

```diff
@@ -2,6 +2,7 @@
 <div style="float: right;">
 	<p><a href="../../system/admin.php?fct=preferences&op=showmod&mod=<{$adminPage.formulizeModId}>"><img src="../images/kedit.png"> <{$smarty.const._AM_HOME_PREF}></a></p>
 	<p style="text-align: right;"><a  href="ui.php?page=import">Import Application (beta!)</a></p>
+    <p style="text-align: right;"></a> <a href="'.XOOPS_URL.'/modules/formulize/admin/ui.php?page=export&aid='.$aid.'"><i class="icon-download"></i> Export (beta!)</a></p>

```

Comment:
Sorry, I see that I made a previous comment on the line for the Export link. The export link is per-application, so it needs to be inside each application block. The link/button to clear templates is per-site, not per-application, so it should be moved here instead.